### ISSUE TITLE
Director Dashboard - Bill per team

### ISSUE THREAD

**Issue by bguerrettaz** [YBNetwork/Issues-Enhancements#1]
Correct director dashboard PayPal account; create mechanism for the director to enter the number of teams and be charged appropriately - estimate: 5-15 hours (still some unknowns here from a code perspective)

**Comment by bguerrettaz** [comment]
Ready to test

**Comment by kirkwhiteman1** [comment]
Tested to the point of NOT paying.  We discussed in e-mails.  Is this how this works - just communicate through here?

**Comment by bguerrettaz** [comment]
Yes, you can reply from your email or reply directly in GitHub (just
remember that if you want to add images or attachments, you'll have to do
it directly in GitHub).

This is totally free, no need to pay for anything.

**Comment by bguerrettaz** [comment]
Any updates on this issue?  Any changes you'd like to see?

**Comment by kirkwhiteman1** [comment]
Right now, this issue #1 looks fine.  I will know more as we approach the next couple weeks as our directors start to post their schedules.  Just make sure that when the director's pay the fee - they have total access to post schedule & results.

**Comment by bguerrettaz** [comment]
Since we're active now, we'll have to make sure to test that functionality
(after directors pay, total access to schedule).

Would you be able to test that with 1 team to make sure?  Maybe from a
personal PayPal to the YB PayPal?

**Comment by bguerrettaz** [comment]
Forgot to mention I have made the change to be able to select up to 350 teams.
